config/envs/default.py

```diff
@@ -93,6 +95,18 @@ class DefaultConfig(object):
     NOTIFICATION_SERVICE_HOST = environ.get("NOTIFICATION_SERVICE_HOST")
     SEND_ENDPOINT = "/send"
     NOTIFY_TEMPLATE_MAGIC_LINK = "MAGIC_LINK"
+    NOTIFICATION_MAGIC_LINK_CONTACT_HELP_EMAIL = (
+        CommonConfig.NOTIFICATION_MAGIC_LINK_CONTACT_HELP_EMAIL
+    )
+    NOTIFICATION_MAGIC_LINK_MAGIC_LINK_URL = (
+        CommonConfig.NOTIFICATION_MAGIC_LINK_MAGIC_LINK_URL
+    )
+    NOTIFICATION_MAGIC_LINK_REQUEST_NEW_EMAIL_URL = (
+        CommonConfig.NOTIFICATION_MAGIC_LINK_REQUEST_NEW_EMAIL_URL
+    )
+    NOTIFICATION_MAGIC_LINK_FUND_NAME = (
+        CommonConfig.NOTIFICATION_MAGIC_LINK_FUND_NAME
```

Comment:
Should this come from the fund store API rather than config?

Reply:
It's not the name itself, but the name of the field for fund name in the notification json. So the literal value is just `fund_name`